tests/cdc-wal2json/000000010000000000000002.sql

```diff
@@ -55,6 +55,14 @@ EXECUTE dc973d3c["11.990000","28814","592","1","3973","11.950000","2022-07-06 12
 PREPARE dc973d3c AS UPDATE "public"."payment_p2022_07" SET "amount" = $1 WHERE "payment_id" = $2 and "customer_id" = $3 and "staff_id" = $4 and "rental_id" = $5 and "amount" = $6 and "payment_date" = $7;
 EXECUTE dc973d3c["11.990000","29136","13","2","8831","11.950000","2022-07-22 16:15:40.797771+00"];
 COMMIT; -- {"xid":501,"lsn":"0/24E7158","timestamp":"2024-02-02 09:33:28.573357+0000"}
+BEGIN; -- {"xid":503,"lsn":"0/24E3598","timestamp":"2024-04-26 09:36:17.632070+0000","commit_lsn":"0/24E3870"}
```

Comment:
It's very strange now to have xid 501, then 503, then 502. That said it could happen in the real world I believe.